advanced_functionality/tensorflow_iris_byom/tensorflow_BYOM_iris.ipynb

```diff
@@ -371,11 +387,11 @@
   }
  ],
  "metadata": {
-  "instance_type": "ml.m5.xlarge",
+  "instance_type": "ml.t3.medium",
   "kernelspec": {
    "display_name": "Python 3 (TensorFlow 2.3 Python 3.7 CPU Optimized)",
    "language": "python",
-   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:us-east-2:429704687514:image/tensorflow-2.3-cpu-py37-ubuntu18.04-v1"
```

Comment:
This change is because of saved settings and will be overridden when cx runs notebook. Can be safely merged. Tested E2E manually